config/authors/jayasimha-raghavan-unskript.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Jayasimha Raghavan",
+  "bio": "",
```

Comment:
Member Of Technical Staff, unSkript, Inc

Reply:
Updated